openqa-trigger-bisect-jobs

```diff
@@ -298,11 +305,20 @@ def main(args):
                 "MAINT_TEST_REPO=",
             ]
         )
+        import pdb
+        #pdb.set_trace()
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except RuntimeError as err:
```

Comment:
I thought it was subprocess.SubprocessError?

Reply:
lets close this as we handle this on https://github.com/os-autoinst/scripts/pull/376#discussion_r1952767628